pandas/core/generic.py

```diff
@@ -5458,13 +5458,63 @@ def asof(self, where, subset=None):
     # Action Methods
 
     _shared_docs['isna'] = """
+        Detect missing values.
+
         Return a boolean same-sized object indicating if the values are NA.
+        NA values, such as None or :attr:`numpy.NaN`, get mapped to True
+        values.
+        Everything else get mapped to False values. Characters such as empty
```

Comment:
get -> gets ?

Reply:
Sry, not my language. Fixed the typos and changed the link as proposed in .notna docstring.